cocos/tween/actions/action-manager.ts

```diff
@@ -381,29 +424,17 @@ export class ActionManager {
         }
     }
 
-    /**
-     * @en
-     * purges the shared action manager. It releases the retained instance. <br/>
-     * because it uses this, so it can not be static.
-     * @zh
-     * 清除共用的动作管理器。它释放了持有的实例。 <br/>
-     * 因为它使用 this，因此它不能是静态的。
-     * @method purgeSharedManager
-     */
-    purgeSharedManager (): void {
-        legacyCC.director.getScheduler().unscheduleUpdate(this);
```

Comment:
There is no reference for `purgeSharedManager`.
And we never add a `update` callback to `Scheduler` for ActionManager.
The internal `ActionManager` is driven by `TweenSystem` in Cocos Creator 3.x.